app/controllers/placements/providers/placements_controller.rb

```diff
@@ -5,8 +5,7 @@ class Placements::Providers::PlacementsController < ApplicationController
   def index
     @subjects = Subject.order_by_name.select(:id, :name)
     @school_types = compact_school_attribute_values(:type_of_establishment)
-    @schools = Placements::School.select(:id, :name)
-    @partner_schools = @provider.partner_schools.select(:id, :name)
+    set_schools
 
     @pagy, @placements = pagy(
       Placements::PlacementsQuery.call(
```

Comment:
We talked about changing this:

```ruby
Placements::PlacementsQuery.call(
  params: filter_form.query_params,
)
```

to:

```ruby
Placements::PlacementsQuery.call(
  params: query_params,
)
```

and introducing a `query_params` method in the controller, so we can include extra context and params to the query object beyond those which exist in the `FilterForm`.

```ruby
def query_params
  {
    filters: filter_form.query_params,
    current_provider: @provider,
  }
end
```

That way, the current provider becomes accessible to the query object.